src/Tests/hooks.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+
+// if some stuffs are needed before/after starting tests,
+export const mochaHooks = {
+  suiteSetup() {
+    const msg = 'All tests start!';
+    console.log(msg);
+    vscode.window.showInformationMessage(msg);
```

Comment:
Can I ask a question?
Could you briefly explain the reason of having this line in addition of L23?
Is Mocha show other messages in a form of `vscode.window.showInformationMessage(msg)` or does Mocha relies on vscode UI (e.g., `vscode.window`)?

Reply:
Mocha itself doesn't rely on vscode. The reason why the code `console.log()` is to show how to add hooks before/after tests. `console.log()` is `console.log()` so that the msg prints on debug console.

Reply:
I was asking why we need to show on console and also on GUI but now I think both are fine. :-)